### ISSUE TITLE
Allow to use existing uv_loop_t instance

### ISSUE THREAD

**Issue by dkrikun** [uNetworking/uWebSockets#627]
Applications might already have a libuv eventloop for its own business.  It would be nice to be able to schedule uWebSockets tasks on the existing instance of uv_loop_t.

**Comment by alexhultman** [comment]
Yep. There is a uv_default_loop() function that returns the default loop. This one is used by default. So Hub::getLoop() will return the same uv_default_loop() by dfault. Otherwise it shold be very easy to pass a loop to the constructor og Hub (with a tiny modification).
